spec/models/profile_spec.rb

```diff
@@ -394,7 +394,7 @@
 
     context 'it notifies the user' do
       let(:profile) do
-        profile = create(:profile, user: user, fraud_review_pending_at: Time.zone.today - 1.day)
+        profile = user.profiles.create(fraud_review_pending_at: Time.zone.today - 1.day)
```

Comment:
as long we we're here, WDYT about switching the date helper methods?
```suggestion
        profile = user.profiles.create(fraud_review_pending_at: 1.day.ago)
```